### PR TITLE
Added Italian language support

### PR DESCRIPTION
**Pull request by ValentinoUberti**
Added Italian language support #33

### CODE REVIEW

**Comment by FoggyMtnDrifter** [comment]
Looks great @ValentinoUberti, but looks like we're missing the changes in `vercel.json`. Could you add that and @hbjydev will authorize a deployment to test before merging.

**Comment by ValentinoUberti** [comment]
> Looks great @ValentinoUberti, but looks like we're missing the changes in `vercel.json`. Could you add that and @hbjydev will authorize a deployment to test before merging.

Done!

File: vercel.json [hunk 1]
```diff
@@ -9,6 +9,8 @@
   "routes": [
     { "handle": "filesystem" },
     { "src": "/es/(.*)", "status": 404, "dest": "/es/404.html" },
+    { "src": "/it/(.*)", "status": 404, "dest": "/it/404.html" },
     { "src": "/(.*)",    "status": 404, "dest": "/404.html" }
+
```
Comment (hbjydev) [inline vercel.json]: Small nitpick, but could you remove this whitespace? Other than that, looks good!

**Review by hbjydev — CHANGES_REQUESTED** [review]

File: vercel.json (hunk 1 above)
Comment (ValentinoUberti) [reply vercel.json]: Done!

**Comment by hbjydev** [comment]
@ValentinoUberti Can you resolve the conflicts? Once that's done I'll merge.

**Review by hbjydev — APPROVED** [review]

**Comment by ValentinoUberti** [comment]
> @ValentinoUberti Can you resolve the conflicts? Once that's done I'll merge.

Fixed

**Pull request merged**